.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
 service-web-store.code-workspace
+
+### Docker Data ###
+docker/data
```

Comment:
In order to persist data between starting/stopping the DB, we mount a local volume that maps to this directory. Postgres stores the DB data in here, but we don't need to check it in.